docker/worker/Dockerfile_x86

```diff
@@ -0,0 +1,31 @@
+#!BuildTag: openqa_worker_x86
```

Comment:
Why is this needed? Do you have a ticket or documentation we can link here? Or a quote from a friendly OBS expert perhaps?

Reply:
That's the tag the container will get.
We have this here too: https://github.com/os-autoinst/os-autoinst/blob/master/docker/travis_test/Dockerfile#L1